kwok/usersignup_test.go

```diff
@@ -0,0 +1,88 @@
+package kwok
```

Comment:
The name of the file suggest it is testing `useContext`, `setContext`, and `setCluster`; but as the title of the PR suggest, it is testing `get-uid-map` using the functions from `usersignup` as pre-configurations. Consider changing it, as it creates confusion.

Reply:
better. is there a reason not to name it `getuidmap_test` or something similar? is it expected to test more things in the future?

Reply:
It runs the get-uid-map script from the script to gather the user signups and passes them as input for user signup tests. Ideally, both are correct.